packages/edit-post/src/components/visual-editor/block-inspector-button.js

```diff
@@ -43,8 +43,8 @@ export function BlockInspectorButton( {
 	};
 
 	const label = areAdvancedSettingsOpened
-		? __( 'Hide Block Settings' )
-		: __( 'Show Block Settings' );
+		? __( 'Hide More Settings' )
+		: __( 'Show More Settings' );
```

Comment:
The label should use sentence case to match other controls.
```suggestion
		? __( 'Hide more settings' )
		: __( 'Show more settings' );
```

Reply:
I wanted to be consisted with other labels in that menu:
![image](https://user-images.githubusercontent.com/1191500/85467010-ee948500-b5aa-11ea-948b-bb770277e86f.png)

Reply:
Oh, well in that case I guess it's fine for now. I guess we can change all of those at once in a separate PR.